e2e-tests/tests/config-router/08-drop-finalizer

```diff
@@ -0,0 +1,5 @@
+apiVersion: ps.percona.com/v1alpha1
+kind: PerconaServerMySQL
```

Comment:
Please remove this file and all '*-drop-finalizer' files from all tests. We have duplicates.

Reply:
Yes, I also saw it, I will remove all of those non-yaml ones.

Reply:
[6382bdd](https://github.com/percona/percona-server-mysql-operator/pull/292/commits/6382bddf6a883b149e58954bdc6350b6114ab4e4)